accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableinstaller/impl/AuthInstallerUserManagerPrefetchingImpl.java

```diff
@@ -61,6 +61,7 @@ public AuthInstallerUserManagerPrefetchingImpl(UserManager delegate, final Value
 
         long startPrefetch = System.currentTimeMillis();
         Iterator<Authorizable> authorizablesToPrefetchIt = delegate.findAuthorizables(new Query() {
+            // all groups and user "anonymous"
```

Comment:
this comment is fully correct, system users are also included

Reply:
Fixed in https://github.com/Netcentric/accesscontroltool/pull/558/commits/52b654000e5b9db417285444e52b9b8a23851ea7.